docs/pages/en-us/Getting-Started.md

```diff
@@ -218,7 +66,7 @@ If you reference the XML schema file from your audit event definition file, then
 
 	<AuditedApplication xmlns="http://www.hpe.com/CAF/Auditing/Schema/AuditedApplication.xsd"
 	                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	                    xsi:schemaLocation="http://www.hpe.com/CAF/Auditing/Schema/AuditedApplication.xsd http://rh7-artifactory.svs.hpeswlab.net:8081/artifactory/policyengine-release/com/hpe/caf/caf-audit-schema/1.1/caf-audit-schema-1.1.jar!/schema/AuditedApplication.xsd">	                   
+	                    xsi:schemaLocation="http://www.hpe.com/CAF/Auditing/Schema/AuditedApplication.xsd https://oss.sonatype.org/content/repositories/releases/com/github/cafaudit/caf-audit-schema/3.0.0/caf-audit-schema-3.0.0.jar!/schema/AuditedApplication.xsd">	                   
 
 Many IDEs and XML editors use the schema file to provide IntelliSense and type-ahead when authoring the definition file.
```

Comment:
Many IDEs and XML editors use the schema file to provide IntelliSense / Auto-Complete when authoring the definition file.